common/protocol/Ola.proto

```diff
@@ -266,6 +266,8 @@ message UniverseInfo {
   required int32 input_port_count = 4;
   required int32 output_port_count = 5;
   required int32 rdm_devices = 6;
+  repeated PortInfo input_ports = 7;
```

Comment:
Presumably that also means the functions in the code that generate them need to check for Has(field_name) too? @simark are you happy to fix these please?